README.md

```diff
@@ -171,7 +171,8 @@ A scale’s domain (the extent of its inputs, abstract values) and range (the ex
 
 * *scale*.**domain** - typically [*min*, *max*], or an array of ordinal or categorical values
 * *scale*.**range** - typically [*min*, *max*], or an array of ordinal or categorical values
-* *scale*.**reverse** - reverses the domain, say to flip the chart along *x* or *y*
+* *scale*.**unknown** - the desired output value (defaults to undefined) for input values outside the domain
```

Comment:
to be more precise unknown only applies to:
- ordinal or categorical scales (not point or band); for values that are outside the domain.
- quantitative scales; for undefined, null or NaN.

Reply:
Good point. Added documentation.